squidDrone/.project

```diff
@@ -1,34 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm missing this file.

Reply:
Readded in [this commit](https://github.com/elheck/Squiddrone/pull/239/commits/6bf0a177d6d45301e222f00792db99885480eedb)